ui_framework/components/modal/modal.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+export function KuiModal({ children, ...rest }) {
+  return (
+    <div className="kuiModal" { ...rest }>
```

Comment:
I mostly dislike the "rest" pattern _unless_ it's for low-level components that are basically just "simple DOM wrappers" where we usually want to push multiple attributes through (like a wrapper around `a`, where we don't actually do anything to `href` etc in the component).

(Though, I've never built components in this way where they are all just "one-line wrappers all the way down", so I don't have any experience in this context to base my opinions on)